code/modules/admin/verbs/buildmode.dm

```diff
@@ -297,3 +298,33 @@
 				if(holder.throw_atom)
 					holder.throw_atom.throw_at(object, 10, 1)
 					log_admin("[key_name(usr)] threw [holder.throw_atom] at [object]",admin_key=key_name(usr))
+
+/verb/load_template()
+	set name = "Load Template"
+	set category = "Special Verbs"
+	if(!usr)
+		return
+	var/list/templates
+	try
+		templates = json_decode(return_file_text("config/templates_list.json"))
+	catch(var/exception/ej)
+		log_debug("Warning: Could not load the templates config as templates_list.json is missing - [ej]")
+		return
+
+	if(!templates)
+		return
+
+	var/turf/T = get_turf(usr)
+	var/name = input(usr, "Which template would you like to load?", "Load Template", null) as null|anything in templates
+	
+	if (!name || !T)
+		return
+
+	var/dmm_suite/maploader = new
+	if (!maploader)
+		log_debug("Error, unable to load maploader in proc load_template!")
+		return
+
+	var/base_dir = "maps/templates/"
+	maploader.load_map(file(base_dir + name), T.x, T.y, T.z, no_changeturf = TRUE)
+	log_and_message_admins("[usr.ckey] has loaded template [base_dir + name] at the coordiantes [T.x], [T.y], [T.z].")
```

Comment:
```suggestion
	log_and_message_admins("[usr.ckey] has loaded template [base_dir + name] at the coordinates [T.x], [T.y], [T.z].")
```

It's also worth making a JMP link for admins to jump to.

Reply:
log_and_message_admins() creates JMP link automatically